app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -385,6 +385,13 @@ class DefaultBrowserToolbarController(
                     BrowserFragmentDirections.actionGlobalHistoryFragment()
                 )
             }
+
+            ToolbarMenu.Item.Downloads -> {
```

Comment:
To avoid the flash on the browser, you can add 	`browserAnimator.captureEngineViewAndDrawStatically {` before navigating like we do for history, bookmarks, etc here